client/src/components/profile/memberscard.js

```diff
@@ -0,0 +1,127 @@
+import withStyles from '../jss'
+import { getMembers, addMember, removeMember } from '../../api/members'
+import { updateAccount } from '../../api/profile'
+import { useState, useCallback, useEffect } from 'preact/hooks'
+import config from '../../../../config/client'
+
+import Form from '../form'
+import { withToast } from '../toast'
+
+const MemberRow = ({ id, name, setMembers }) => {
+  const handleDelete = useCallback(() => {
+    removeMember({ id })
+      .then(() => {
+        setMembers(members => members.filter(a => a.id !== id))
+      })
+  }, [id, setMembers])
+
+  return (
+    <div class='u-vertical-center' style='width: 100%; display: flex; justify-content: space-between;' key={id}>
+      <p class='u-no-margin'>{name}</p>
+      <div class='btn-container u-vertical-center'>
+        <input onClick={handleDelete} type='submit' class='btn-tiny btn-danger u-no-margin' value='Delete' />
+      </div>
+    </div>
+  )
+}
+
+const MembersCard = withStyles({
+
+}, withToast(({ classes, division: originalDivision, toast }) => {
+  const [name, setName] = useState('')
+  const handleNameChange = useCallback(e => setName(e.target.value), [])
+
+  const [email, setEmail] = useState('')
+  const handleEmailChange = useCallback(e => setEmail(e.target.value), [])
+
+  const [grade, setGrade] = useState('')
+  const handleGradeChange = useCallback(e => setGrade(e.target.value), [])
+
+  const [buttonDisabled, setButtonDisabled] = useState(false)
+
+  const [division, setDivision] = useState(originalDivision)
+  const handleDivisionChange = useCallback(e => {
+    e.preventDefault()
+
+    const division = e.target.value
+
+    updateAccount({ division })
+      .then(({ error, data }) => {
+        if (error) {
+          toast({ body: error, type: 'error' })
+        } else {
+          setDivision(data.division)
+          toast({ body: 'Division successsfully updated' })
```

Comment:
```suggestion
          toast({ body: 'Division successsfully updated', type: 'success' })
```

Reply:
Cirrus 'success' styling looks really bad on toasts, so I'm just using the default to denote "success".